assets/src/components/detours/diversionPage.tsx

```diff
@@ -3,9 +3,9 @@ import { DiversionPanel, DiversionPanelProps } from "./diversionPanel"
 import { DetourMap } from "./detourMap"
 import { Shape } from "../../schedule"
 import { useDetour } from "../../hooks/useDetour"
+import { ListGroup } from "react-bootstrap"
 
 export const DiversionPage = ({
-  directions,
   missedStops,
```

Comment:
Not  for this PR, but will `missedStops` also get dropped from the public contract for this component? That also seems like something that might get computed _within_ `DiversionPage` rather than outside of it.

Reply:
I'm still on the fence about where stuff should live ([to use a MVC analogy] right now, I see `useDetour` as like, the model, `DiversionPage` is acting as a controller in a way, but `DiversionPage` is also the view)

But I think that `missedStops` will _likely_ be removed, still mulling it over though.

Reply:
I imagine that sometime soonish, it'll make sense to give `DiversionPage` a treatment similar to the one you gave to `DetourMap`, so that it's pretty much only dealing with the rendering side of the world (and noticing that there's input, I guess?). 

I almost would rather see that as a follow-up PR, since as it stands, this doesn't look too painful, and I like the idea of keeping feature work and refactor work separate. I don't feel too strongly about it though, if doing that refactor here feels _right_ to you, then I have no objection.